app/models/megaphone/podcast.rb

```diff
@@ -54,7 +54,7 @@ def self.attributes_from_feed(feed)
       podcast = feed.podcast
       itunes_categories = feed.itunes_categories.present? ? feed.itunes_categories : podcast.itunes_categories
       {
-        title: feed.title || podcast.title,
+        title: podcast.title,
```

Comment:
the feed title is always "Megaphone Feed", so use the podcast's

Reply:
I do wonder if we should have distinguished "feed title override" vs "feed label".  Where the label is just for the Feeder UI, so you can tell them apart.

Ah well, problem for our future selves.